shell/platform/tizen/tizen_embedder_engine.cc

```diff
@@ -16,8 +16,37 @@
 // Unique number associated with platform tasks.
 static constexpr size_t kPlatformTaskRunnerIdentifier = 1;
 
+static std::string GetDeviceProfile() {
+  char* feature_profile;
+  system_info_get_platform_string("http://tizen.org/feature/profile",
+                                  &feature_profile);
+  std::string profile(feature_profile);
+  free(feature_profile);
+  return profile;
+}
+
+static std::string GetDeviceModelName() {
+  char* system_model_name;
+  system_info_get_platform_string("http://tizen.org/system/model_name",
+                                  &system_model_name);
+  std::string model_name(system_model_name);
+  free(system_model_name);
+  LoggerD("model_name : %s", model_name.data());
+  return model_name;
+}
+
+static double GetDeviceDpi() {
+  int feature_dpi;
+  system_info_get_platform_int("http://tizen.org/feature/screen.dpi",
+                               &feature_dpi);
+  return (double)feature_dpi;
+}
+
 TizenEmbedderEngine::TizenEmbedderEngine(
-    const FlutterWindowProperties& window_properties) {
+    const FlutterWindowProperties& window_properties)
+    : device_profile(GetDeviceProfile()),
+      device_model_name(GetDeviceModelName()),
+      device_dpi(GetDeviceDpi()) {
```

Comment:
This looks a bit messy. You may initialize the values inside the constructor.

Reply:
The compiler does not allow this for const member